src/server/ssrServer.js

```diff
@@ -121,12 +123,25 @@ export function createApp({ enablePostToModuleRoutes = false } = {}) {
   }
 
   // https://expressjs.com/en/guide/error-handling.html
+  router.use(serverError); // Note: should be quickly moved to Fastify
+
+  return router;
+};
+
+export async function createApp(opts = {}) {
+  const fastify = Fastify(opts);
+
+  await fastify.register(fastifyExpress);
+
+  fastify.express.disable('x-powered-by');
+  fastify.express.disable('e-tag');
+
+  fastify.use(makeExpressRouter());
 
-  app.use(serverError);
+  // TODO: Needs refactoring (priority)
```

Comment:
Is this TODO for this PR?

Reply:
no, this is for when we migrate the express routes to fastify. Server errors are being handled from the express router but we still need a 404 handler on the top level